docs/operator-guides/security.md

```diff
@@ -147,7 +147,7 @@ Airbyte Cloud supports [user management](https://docs.airbyte.com/cloud/managing
 Our compliance efforts for Airbyte Cloud include:
 
 - SOC 2 Type II assessment: An independent third-party completed a SOC2 Type II assessment and found effective operational controls in place. Independent third-party audits will continue at a regular cadence, and the most recent report is available upon request.
-- ISO 27001 certification: We are currently pursuing ISO 27001 certification and will continue to align the evolution of our security program with its standards as we grow.
+- ISO 27001 certification: We received our ISO 27001 certification in November 2022. If you would like a copy of the certification, contact us at security@airbyte.io 
```

Comment:
@Patsy-Airbyte is security@airbyte.io the email that you would like users to contact if they want a copy of the certification?

Reply:
Could we use the same language as the SOC2 section? "A copy of the certificate is available upon request."

Reply:
@Patsy-Airbyte Edited :)